rsciio/tests/test_hamamatsu.py

```diff
@@ -399,3 +400,30 @@ def test_metadata(self):
     def test_data(self):
         expected_data = [9385, 8354, 7658]
         np.testing.assert_allclose(self.s.isig[:3, 0].data, expected_data)
+
+
+class TestOther:
+    @classmethod
+    def setup_class(cls):
+        cls.s = hs.load(testfile_xaxisother_path, reader="Hamamatsu")
+
+    @classmethod
+    def teardown_class(cls):
+        del cls.s
+        gc.collect()
+
+    def test_xaxisother(self):
```

Comment:
Since there is only test in this class, would it be more simple to use a function instead of a class?

Reply:
As there is a class per file used for the tests of this reader, I wanted to stick to that pattern.